configurations/tablets-initial-32.yaml

```diff
@@ -1,5 +1,5 @@
-append_scylla_yaml: |
-  experimental_features:
-    - tablets
-    - consistent-topology-changes
+experimental_features:
+  - tablets
+  - consistent-topology-changes
```

Comment:
btw. recently these got removed from experimental.
To avoid hassle with merge conflicts later I'd propose to switch it to `enable_tablets: true` right away

Reply:
Maybe it's better to do both changes - cleanup of usages of these 2 features as experimental and add `enable_tablets` flag to scylla.yaml - in a separate PR?
For `consistent-topology-changes` there is a test config, that sets only this option - enables as experimental feature - `consistent-topology-changes`. That test config is used in 31 places in the SCT.
For `tablets` - it is also used in 2 test configs that do only that - enable it as experimental feature. And these 2 configs are used in 11 places in SCT.

Reply:
yes, thinking again we should split it to separate PR as it will also require change in `ScyllaYaml` - too much for this change.